Source/WebCore/platform/TextureMapper.cmake

```diff
@@ -69,6 +71,13 @@ if (USE_COORDINATED_GRAPHICS)
         platform/graphics/texmap/coordinated/TiledBackingStoreClient.h
     )
 
+    if (USE_GSTREAMER)
+        list(APPEND WebCore_SOURCES
+            platform/graphics/texmap/coordinated/CoordinatedPlatformLayerBufferHolePunch.cpp
```

Comment:
This is also needed by MediaPlayerPrivateHolePunch, which doesn't require GStreamer